project/app/utils/visualizations/histogram.py

```diff
@@ -0,0 +1,80 @@
+# Function to produce a histogram that shows the distribution of the scores
+# from the student's grade level for the current week
+# Plots a vertical line of the students most recent score for comparison
+
+# Imports
+import plotly.express as px
+
+
+def histogram(grade_list, student_info):
+    '''
+    Plotly histogram of all of the submission scores for the student's grade
+    for the current week
+    Plots a vertical line for the student's most recent score so they can see
+    how it compares to the rest of the grade
+
+        Input: List of the current week's scores for the specific grade level
+        Student information in a list in this order: 
+            [grade_number, student_name, student_score]
+
+        Output: Plotly JSON that can be passed to the web to display on the
+        parent dashboard
+    '''
+    # Dynamic variables to use for labels on the plot
```

Comment:
Also for [PEP8 compliance](https://www.python.org/dev/peps/pep-0257/), there should be one blank line after the docstring as well.